resource-definitions/template-driver/namespace/short-namespace.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: entity.humanitec.io/v1b1
+kind: Definition
+metadata:
+  id: custom-namespace
+entity:
+  name: custom-namespace
+  type: k8s-namespace
+  driver_type: humanitec/template
+  driver_inputs:
+    values:
+      templates:
+        # Here the namespace name is shortened to be a maximum of 17 characters,
+        # no matter how long the app and env name might be.
+        init: |
+          name: name: {{ substr 0 8 "${context.env.id}" }}-{{ substr 0 8 "${context.app.id}" }}
```

Comment:
This looks wrong. There are two `name` properties in line?

Also, would suggest this instead:
```
name: {{ trunc 8 "${context.env.id}" }}-{{ trunc 8  "${context.app.id}" }}
```